packages/db/schema.prisma

```diff
@@ -436,6 +437,17 @@ model BetaTokens {
   @@index([userId])
 }
 
+model Donations {
+  id                      Int     @id @default(autoincrement())
+  user                    User    @relation(fields: [userId], references: [id])
+  userId                  String
+  amount                  Decimal @db.Decimal(9, 2)
+  stripeCheckoutSessionId String  @unique
+  createdAt DateTime? @default(now())
+
```

Comment:
Before pushing schema id double check there is nothing else we want from stripe API.

like country and we could show where donations are coming from.